Makefile

```diff
@@ -245,6 +245,8 @@ pkg/qrexec-dom0: pkg/qrexec-lib pkg/xen-tools eve-qrexec-dom0
 	@true
 pkg/qrexec-lib: pkg/xen-tools eve-qrexec-lib
 	@true
+pkg/rsyslog: pkg/gpt-tools eve-rsyslog
```

Comment:
Why does it depend on gpt-tools?

Reply:
We pass the current image partition as env variable to rsyslogd. IMGP=$(zboot curpart).
So, we'll need the gpt-tools for zboot command.

Reply:
Actually, as @nordmark pointed out -- I'd rather avoid creating more container dependencies at this point. This is super brittle and leads to a lot of build issues.

@gkodali-zededa in this particular case -- I'd start with NOT having that dependency and then solving the issue of determining the partition via the /persist untangling container we've discussed.

So in summary -- can you please update this particular PR to not depend on gpttools (even thought it'll kill your IMGP=$(zboot curpart) for now -- but we will bring it back once we do the /persist management untangling -- btw for now just set it statically to IMGA).

Makes sense?

Reply:
Understood. I'll remove the gpt-tools dependency.